Tic_Tac_Toe.py

```diff
@@ -0,0 +1,65 @@
+def main():
+    print("=" * 30)
+    print("""Welcome to Tic Tac Toe
+GAME RULES:
+Each player can place one mark (or stone) per turn on the 3x3 grid
+The WINNER is who succeeds in placing three of their marks in a
+* horizontal,
+* vertical or
+* diagonal row
+Let's start the game...""")
+    board = [str(i + 1) for i in range(0, 9)]
+    player = "x"
+    print_board(board)
+    board = [" " for i in range(0, 9)]
+    while not evaulate_board(board,player):
+        if player == "x":
+            player = "o"
+        else:
+            player = "x"
+        change_board(board,player,move=get_move(player))
+        print_board(board)
+    print("=" * 40)
+    print(f"Congratulations, the player '{player}' WON!")
+
+
+def change_board(board,player,move):
+    while move > 8:
+        input("This position does't exist, press 'Enter' to try again...")
+        move = get_move(player)
+    while board[move] != " ":
+        input("This position is not free, press 'Enter' to try again...")
+        move = get_move(player)
+    board[move] = player
+
+def get_move(player):
+    print("=" * 40)
+    move = input(f"Player {player} | Please enter your move number: ")
+    if not move.isdigit():
+        move = 10
+        return move
```

Comment:
Rather than returning a custom value that you know that should not happen (what if we decide to expand the tic-tac-toe board to 4x4 or to user-defined number?). If you want to return something different than expected result return `None`.